src/web/client/dal/remoteFetchProvider.ts

```diff
@@ -409,7 +415,8 @@ async function createFile(
     mappingEntityFetchQuery: string | undefined,
     entityId: string,
     dataverseOrgUrl: string,
-    portalsFS: PortalsFS
+    portalsFS: PortalsFS,
+    entityFileMap: Map<string, string[]>
```

Comment:
why do we need this entityId map - at a time this call will come for just one entityId and the createVirtual file call will happen for just one of the URI (in case of webpage) associate with an entityId